components/feature/pwa/src/main/java/mozilla/components/feature/pwa/intent/WebAppIntentProcessor.kt

```diff
@@ -39,23 +40,25 @@ class WebAppIntentProcessor(
      * A custom tab config is also set so a custom tab toolbar can be shown when the user leaves
      * the scope defined in the manifest.
      */
-    override suspend fun process(intent: Intent): Boolean {
+    override fun process(intent: Intent): Boolean {
         val url = intent.toSafeIntent().dataString
 
         return if (!url.isNullOrEmpty() && matches(intent)) {
-            val webAppManifest = storage.loadManifest(url) ?: return false
+            runBlocking {
+                val webAppManifest = storage.loadManifest(url) ?: return@runBlocking false
```

Comment:
```suggestion
            val webAppManifest = runBlocking { storage.loadManifest(url) } ?: return false
```

Reply:
@NotWoods , I had to put the `return@runBlocking` since everytime I tried to run I got the "return is not allowed here` complaint, which is why I just did `reutnr@RunBlocking` 🤔

Reply:
Rather than wrapping the entire method with `runBlocking`, you could just wrap the `loadManifest` function. I think return will be fine then too.

Reply:
Ah you were right. Thanks for pointing that out 😄